vue-app/src/components/Cart.vue

```diff
@@ -289,7 +285,17 @@
           {{ tokenSymbol }}
         </div>
       </div>
-      <!-- TODO: reallocation bar -->
+      <div
+        v-if="$store.getters.isRoundJoinOnlyPhase && $stores.state.currentRound"
```

Comment:
```suggestion
        v-if="$store.getters.isRoundJoinOnlyPhase && $store.state.currentRound"
```

Reply:
Also - I don't think this 2nd check is needed. `isRoundJoinOnlyPhase` already checks for `!!state.currentRound`.

Reply:
That makes sense, I believe I added this in when stuff wasnt rendering, but forgot to remove as it wasnt the problem. Thanks for the flag.